examples/chemistry/brusselator_bounded.jl

```diff
@@ -6,12 +6,14 @@ using Catlab.Graphics
 using CombinatorialSpaces
 using CombinatorialSpaces.ExteriorCalculus
 using ComponentArrays
+using DiagrammaticEquations
+using DiagrammaticEquations.Deca
 using Decapodes
 using MultiScaleArrays
 using MLStyle
 using OrdinaryDiffEq
 using LinearAlgebra
-using GLMakie
+using CairoMakie
```

Comment:
Maybe we should use 

```
using CairoMakie
import CairoMakie: wireframe [all the other methods we use]
```

and then we won't have to use the CairoMakie namespace in the code. Then if we switch back to GL, we can just change it at the top of the file.